### DISCUSSION TITLE
管理后台的代码再哪儿下载喃？

### DISCUSSION THREAD

**Discussion by bobtmee** [walinejs/waline#1554] [Q&A] [answered]
管理后台的代码再哪儿下载喃？ 想自定义管理后台样式

**Comment by lizheming** [comment, marked as answer]
可以通过代理 [rewrites](https://vercel.com/docs/project-configuration#project-configuration/rewrites) 重定向页面到自己写的 HTML，copy 下原始的 HTML 页面然后自己加样式
